cmake/modules/AddSwift.cmake

```diff
@@ -580,6 +581,9 @@ endfunction()
 # IS_STDLIB
 #   Install library dylib and swift module files to lib/swift.
 #
+# MODULE_DIR
+#   Directory to install .swiftmodule and .o file in.
```

Comment:
I was mirroring the comment for `MODULE_DIR` here, but you're right `install` is the wrong word. https://github.com/apple/swift/blob/17f619793829b1ccc351b956c6b8f2ad618dc16f/cmake/modules/SwiftSource.cmake#L149-L150